test/performance/README.md

```diff
@@ -49,6 +49,52 @@ if err != nil {
 }
 ```
 
+## Zipkin trace
+
+Zipkin tracing can be enabled if needed by the performance test during setup. Once it is setup, all requests made
```

Comment:
can we add sample code on how to do this?

```go
perfClients, err := Setup(t, EnableZipkinTracing)
```

Reply:
Updated to include sample code.